test/seed.spec.js

```diff
@@ -2525,10 +2525,10 @@ describe("> Contract: Seed", () => {
           // Reach softCap and endTime
           await convertSeedToComplete(Seed_shortTipVesting);
 
-          // Admin should have no Seed tokens before retrieving them
+          // treasury should have no Seed tokens before retrieving them
```

Comment:
fyi VSCode has a feature to search and replace by casing, so "Admin" would have been changed to "Treasury"

![image](https://user-images.githubusercontent.com/30693990/203310643-04198162-37fe-4fae-8759-a8ee2f96f63f.png)

Reply:
Thanks! I actually do know this but because I didn't want to replace all the `admin` with `treasury`, I actually did it manually.